cli/command/utils.go

```diff
@@ -78,7 +78,10 @@ func PrettyPrint(i any) string {
 
 type PromptError error
```

Comment:
Looks like this type was introduced in 2ae903e86cab51f694c819721cdfdf5eec693720, but I don't see it being used as sentinel error; is there a reason we introduced a "special" type for this? We should be careful introducing special types, as (especially when exported) they will add to our public API, and may lead to expectations of them having special meaning.

Where possible, limit to `errdefs` interfaces / definitions, which allow handling specific error "classes" in most cases, and reserve special types where specific handling _must_ be used.

Reply:
This makes sense, it's a habit of mine to "organize" errors under a type. I suppose in this case I can remove it and just use the exported variable instead.